area.java

```diff
@@ -1,9 +1,19 @@
 import java.util.*;
 
-class Area {
+
+class Main {
```

Comment:
I think your class should have the same name as the file, so `Area` - you were right before. Also, I believe that the name of the file should be capitalized - so `Area.java`.

Reply:
alright I will do that right now